apps/vr-tests-react-components/src/stories/DatePickerCompat.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { DatePicker as DatePickerBase, DateRangeType } from '@fluentui/react-datepicker-compat';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../utilities/TestWrapperDecorator';
+import type { DatePickerProps } from '@fluentui/react-datepicker-compat';
+
+const DatePicker = (props: DatePickerProps) => {
+  const today = new Date('3/15/2023');
+  return (
+    <div style={{ width: '500px', height: '330px', padding: '10px' }}>
+      <DatePickerBase value={today} today={today} {...props} />
+    </div>
+  );
+};
+
+storiesOf('DatePicker Compat', module)
+  .addDecorator(TestWrapperDecorator)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .click('.datepicker-input')
+        .snapshot('opened', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory('default', () => <DatePicker input={{ className: 'datepicker-input' }} />, {
```

Comment:
Do you need `input={{ className: 'datepicker-input' }}` for all of the ones below, to make the `click('.datepicker-input')` step work?

Or if you don't want the steps to apply to all stories, you could have a separate `storiesOf` with the same name `'DatePicker Compat'`, but without the story steps. See [Radio.stories.tsx](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Radio.stories.tsx) for an example.